app/views/users/show.html.erb

```diff
@@ -1 +1,31 @@
-<h2>User information and posts</h2>
\ No newline at end of file
+<div class="users-container">
+  <div class="user-card">
+    <div class="user-photo">
+      <%= image_tag @user.photo, alt: "#{ @user.name }'s photo" %>
+    </div>
+    <div class="user-details">
+      <%= link_to @user.name, user_path(@user) %>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+</div>
+
+<div class="user-bio">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+<div class="user-posts">
+  <% @posts.first(3).each do |post| %>
+    <div class="post">
+      <h3><%= post.title %></h3>
+      <p><%= post.text %></p>
+      <div class="cmt-like">
+        <span>Comments: <%= post.comments_counter %></span>
+        <span>Likes: <%= post.likes_counter %></span>
+      </div>
+    </div>
+  <% end %>
+
+  <%= link_to "See all posts", user_posts_path(@user), class: 'button', remote: true %>
```

Comment:
- [ ] Your app is rendering users first 3 post! Over here, we actually want you to use the `#recent_post` method (you created in previous task) to render the user's **3 most recent posts** instead, which your code should be similar to:

```ruby
# app/views/users/show.html.erb

[...]

<% @user.recent_post.each do |post| %>

<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>

<% end %>

<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>
```

**Your app rendered 3 first post instead of 3 recent posts:**
![image](https://github.com/gilberthappi/Blog-app/assets/12745474/6f2179cf-f073-4b47-8b44-c2b5e577792b)